build/build.proj

```diff
@@ -36,6 +37,14 @@
              Targets="Build"
              Properties="$(CommonMSBuildGlobalProperties)"
              />
+
```

Comment:
What does this have to do with RestorePackages? This is building a nuget package - the reverse. :) 

I would put this in its own target.